unotest/runtests.jl

```diff
@@ -0,0 +1,135 @@
+# Copyright (c) 2018-2024: Charlie Vanaret and contributors
+#
+# Use of this source code is governed by an MIT-style license that can be found
+# in the LICENSE.md file or at https://opensource.org/licenses/MIT.
+
+# For help with this file, please contact `@odow` on GitHub.
+
+using Test
+
+import AmplNLWriter
+import MathOptInterface as MOI
+import MINLPTests
+
+"""
+    Optimizer()
+
+Create a new `AmplNLWriter.Optimizer` object that uses Uno as the backing
+solver.
+"""
+function Optimizer()
+    options = String["logger=SILENT"]
+    return AmplNLWriter.Optimizer("uno_ampl", options)
```

Comment:
This `uno_ampl` string needs to be changed to the path of wherever we end up compiling it.